uktrade/great-cms
# GP2-665: Wagtail Transfer - easy local setup skeleton

**This PR is for sub-tasks targetting a main ticket PR, so feel free to merge if you approve** 

This changeset drops in a basic configuration of [Wagtail-Transfer](https://wagtail.github.io/wagtail-transfer/) for LOCAL development/testing only, for now -- fuller configuration (and more documentation) will come with other tickets.

It also updates the makefile, to make running a second local environment a little easier, I hope.

I would welcome some close eyes on the changes I've made, which I don't believe break the default behaviour of the site, and also someone to test-drive my local setup instructions. DM me if you have any questions.

 - [X] Ticket exists in Jira 
 - [X] New env var added to all three deployed envs
 - [X]  How to test locally, including how to set up appropriate data

## To test

See docs/wagtail_transfer.rst -- after that is done you should be able to at least see the W-T UI in the admin, if you click the `Import` option in the sidebar. There's no need to try a transfer at this stage, but you're welcome to try it out, moving edits published on your local 8020 site to your (new) local 'transfer target' site on 8030.
